draftfast/optimizer.py

```diff
@@ -107,7 +113,7 @@ def _set_player_constraints(self):
             if lb > ub:
                 raise InvalidBoundsException
 
-            if p.multi_position:
+            if p.multi_position or self.showdown:
```

Comment:
@sharkiteuthis I *think* this is the same as what was happening with no_reuse_captain and set_single_captain (I believe that one is already handled by the position constraints that look to the rule set).